src/simplebar.js

```diff
@@ -188,6 +188,8 @@ export default class SimpleBar {
 
         this.el.insertBefore(this.trackX, this.el.firstChild);
         this.el.insertBefore(this.trackY, this.el.firstChild);
+
+		this.__isDomInitialized = true;
```

Comment:
There is 2 `_` on this variable. It's `__isDomInitialized` instead of `_isDomInitialized`.